packages/specs/schema/src/decorators/common/groups.ts

```
@@ -34,33 +31,25 @@ export function Groups<T>(groupName: string, groups: string[]): ParameterDecorat
-      case DecoratorTypes.PARAM:
-        let groupsName = "";
+    if (entity.is(DecoratorTypes.PARAM)) {
+      let groupsName = "";
 
-        if (groups.length == 2 && isArray(groups[1])) {
-          groupsName = groups[0];
-          groups = groups[1];
-        }
+      if (groups.length == 2 && isArray(groups[1])) {
+        groupsName = groups[0];
+        groups = groups[1];
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)